### ISSUE TITLE
Strip parameters when identifying images or videos

### ISSUE THREAD

**Issue by bu5hm4nn** [mikedilger/gossip#362]
<img width="805" alt="Screenshot 2023-04-28 at 17 48 01" src="https://user-images.githubusercontent.com/3328670/235270873-cf28d157-55df-460d-861a-78d179382fb5.png">
nevent1qqsvdy7jxduj6hdlsnt3kmvyw0dz0z8np8guulvvtepa26f3zxx7zyqppemhxue69uhkummn9ekx7mp0ywdhll

**Comment by mikedilger** [comment]
That doesn't just change the detection as video, it changes the link that gets sent to the server. In the example it probably doesn't matter but it will in other cases.

I would instead use the Url crate (we don't have it in Cargo.toml, but nostr-types pulls it in already) and do something like
````
let url = url::Url::parse(link.trim())?;
if url.path()  blah blah blah
```

**Comment by bu5hm4nn** [comment]
Shoot you're right. I guess that's what happens when you write code while having a fever lol. Will fix.

**Comment by bu5hm4nn** [comment]
Fixed with #364
